File: runelite-client/src/main/java/net/runelite/client/plugins/autoVorkath/AutoVorkathConfig.java
package net.runelite.client.plugins.autoVorkath;

import net.runelite.client.config.Config;
import net.runelite.client.config.ConfigGroup;
import net.runelite.client.config.ConfigItem;

@ConfigGroup("AutoVorkath")
public interface AutoVorkathConfig extends Config {

    @ConfigItem(
            keyName = "crossbow",
            name = "Crossbow",
            description = "Choose your crossbow",
            position = 0
    )
    default CROSSBOW CROSSBOW() {
        return CROSSBOW.ARMADYL_CROSSBOW;
    }

    @ConfigItem(
            keyName = "slayersStaff",
            name = "Slayers Staff",
            description = "Choose your slayers staff",
            position = 1
    )
    default STAFF SLAYERSTAFF() {
        return STAFF.SLAYER_STAFF;
    }

    @ConfigItem(
            keyName = "teleport",
            name = "Teleport",
            description = "Choose your teleport",
            position = 2
    )
    default TELEPORT TELEPORT() {
        return TELEPORT.CONSTRUCT_CAPE_T;
    }

    @ConfigItem(
            keyName = "rigour",
            name = "Rigour",
            description = "Activate Rigour?",
            position = 3
    )
    default boolean ACTIVATERIGOUR() { return true; }
}

File: runelite-client/src/main/java/net/runelite/client/plugins/autoVorkath/AutoVorkathConfig.kt
package net.runelite.client.plugins.autoVorkath

import net.runelite.client.config.Config
import net.runelite.client.config.ConfigGroup
import net.runelite.client.config.ConfigItem

@ConfigGroup("AutoVorkath")
interface AutoVorkathConfig : Config {
    @ConfigItem(
        keyName = "crossbow",
        name = "Crossbow",
        description = "Choose your crossbow",
        position = 0
    )
    fun CROSSBOW(): CROSSBOW? {
        return CROSSBOW.ARMADYL_CROSSBOW
    }

    @ConfigItem(
        keyName = "slayersStaff",
        name = "Slayers Staff",
        description = "Choose your slayers staff",
        position = 1
    )
    fun SLAYERSTAFF(): STAFF? {
        return STAFF.SLAYER_STAFF
    }

    @ConfigItem(
        keyName = "teleport",
        name = "Teleport",
        description = "Choose your teleport",
        position = 2
    )
    fun TELEPORT(): TELEPORT? {
        return TELEPORT.CONSTRUCT_CAPE_T
    }
}
